jetson/learn_features.py

```diff
@@ -10,15 +10,16 @@
 from torch.utils.tensorboard import SummaryWriter
 from torchvision import datasets
 
-#from goggles.goggleClassifier import MapDataset, train_model
-
+# 80/20 training/validation split
 VAL_SPLIT = 0.2
 
 
 def tensor_loader(path):
+    """Required for the custom dataset created in load_data."""
```

Comment:
this tells me why this function is needed not what it does. I need both as a developer to use it properly